client/src/App.js

```diff
@@ -6,18 +6,22 @@ import SignIn from './pages/SignIn';
 import SignUp from './pages/SignUp';
 import ShoeDetail from './pages/ShoeDetail';
 import ApparelDetail from './pages/ApparelDetail';
+import Header from './components/Header';
 import { Routes, Route } from 'react-router-dom';
 
 export default function App() {
   return (
-    <Routes>
-      <Route index path="/" element={<Category />} />s
-      <Route path="shoes" element={<Shoes />} />
-      <Route path="apparels" element={<Apparels />} />
-      <Route path="sign-in" element={<SignIn />} />
-      <Route path="sign-up" element={<SignUp />} />
-      <Route path="shoes/:listingId" element={<ShoeDetail />} />
-      <Route path="apparels/:listingId" element={<ApparelDetail />} />
-    </Routes>
+    <div>
+      <Header />
```

Comment:
This is fine, but using `Outlet` gives you more flexibility when designing your layout. Something to think about in the future.